src/components/Rate/index.tsx

```diff
@@ -0,0 +1,212 @@
+import Box from '@mui/material/Box'
+import { useNavigate, useParams } from 'react-router-dom'
+import Typography from '@mui/material/Typography'
+import { TextField, Button, FormControl, MenuItem, Rating, FormHelperText } from '@mui/material'
+import IHomeProps from '../../containers/Form/types'
+import styles from '../../containers/Form/styles'
+import { Controller, useForm } from 'react-hook-form'
+import { useCreateClaim } from '../../hooks/useCreateClaim'
+import Tooltip from '@mui/material/Tooltip'
```

Comment:
Same list it before local imports

Reply:
@AmrNabel this is good idea!  please do in future PR